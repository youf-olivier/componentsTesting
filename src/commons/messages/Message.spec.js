import React from "react";
import { render } from "@testing-library/react";
import Component from "./Message";

describe("Message test suite", () => {
  const message = "Une erreur est survenue";

  it("renders correctly", () => {
    const { debug } = render(
      <Component message={message} onClose={() => {}} />
    );
    debug();
  });
});
