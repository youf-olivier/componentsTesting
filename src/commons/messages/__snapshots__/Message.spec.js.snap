// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message test suite renders correctly 1`] = `
<DocumentFragment>
  <div
    class="error-message"
  >
    <span>
      Une erreur est survenue
    </span>
    <button
      class="error-message__close"
    >
      X
    </button>
  </div>
</DocumentFragment>
`;
